build.gradle

```diff
@@ -104,6 +104,38 @@ task exportSampleSphereBlast {
     }
 }
 
+task buildAndRunInstantAppTest {
+    description 'Creates the instant app test project, then builds and runs in instant mode'
+    def buildAndRunFunction = 'GooglePlayInstant.Editor.AndroidInstantTest.BuildAndRunTestProject'
+    def buildPath = "${playInstantBuildPath}/testproject"
+    def logFile = file("testproject.log").getPath()
+    doFirst {
+        copyFiles(buildPath, [])
```

Comment:
I don't think we need unit tests for this build: ['Tests']

Reply:
I was including it so that AndroidInstantTest.cs would be included but the better solution is to rename and move that script out of that folder.

Reply:
Ack. My comment was made assuming that you'd take my suggestion to move your builder class. That's my bad - I wasn't clear.